src/ng/cacheFactory.js

```diff
@@ -223,6 +223,8 @@ function $CacheFactoryProvider() {
             delete lruHash[key];
           }
 
+          if (!(key in data)) return;
```

Comment:
`createMap` is defined in src/Angular.js. It just performs `Object.create(null);` to build an object with a null prototype